website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -1,4 +1,4 @@
----
+ ---
```

Comment:
Once this is ready to merge doc changes will also need to be reflected in the `website/content/en/docs` path. No rush to add them there, they'll just have to be before merge.

Reply:
I see, is it to version `v0.33`?

Reply:
Since this is a new feature it wouldn't go in until the next minor version bump so no. There should be two paths where doc changes are reflected, `website/content/en/preview` (what you have now), and `website/content/en/docs`.

Reply:
Sorry, I was mistaken here. I thought `website/content/en/docs` is an alias for preview but it's actually an alias for the current version (v0.33). You only need the changes in preview.